src/Fhir.Anonymizer.Core/Extensions/ElementNodeNavExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Model;
+using Hl7.FhirPath;
+using Hl7.FhirPath.Expressions;
+
+namespace Fhir.Anonymizer.Core.Extensions
+{
+    public static class ElementNodeNavExtensions
+    {
+        internal static readonly string s_containedNodeName = "contained";
+        internal static readonly string s_entryNodeName = "entry";
+        internal static readonly string s_resourceNodeName = "resource";
```

Comment:
Not very sure where do we put these constants? It seems they are used in both extensions.

Reply:
Yes, move to Constants